src/account/serialization.ts

```diff
@@ -135,7 +135,7 @@ export const toOperationRaw = (
     blockHeight,
     extra: e,
     date: date.toISOString(),
-    value: value.toString(),
+    value: value.toFixed(),
```

Comment:
could you explain the reason to switch? Thanks

Reply:
Yes, of course. I changed that one to avoid using scientific notation, as it mentions [here](https://github.com/MikeMcl/bignumber.js/#use).

<img width="902" alt="image" src="https://user-images.githubusercontent.com/10941036/133795745-2025ac34-35ba-4545-a2cc-73cdacb4920f.png">

Reply:
wow good to know. thanks.